spec/specs/useDomMeasure.spec.tsx

```diff
@@ -11,7 +12,7 @@ describe("domMeter", () => {
             width: 2,
             height: 6
         };
-        const container = renderHook(useMeter, {current: null});
+        const container = renderHook(useMeter, {current: undefined} as { current: b.IBobrilCacheNode | undefined });
```

Comment:
Navrhuju pro to udelat pomocny typ, Ref =>
```
export type Ref<T> = {
  current: T | undefined;
}
```
pokud chceme explicitne inicializovat na ten undefined. Za me by stacilo
```
export type Ref<T> = {
  current?: T;
}
```

Reply:
přidal jsem, ten current tam musí být kvůli ref v renderHooku